### PR TITLE
Split out signature from metadata group, adjust all images

### PR DESCRIPTION
**Pull request by pmai**

### CODE REVIEW

**Comment by pmai** [comment]
MAP SSP Meeting 2024-11-08: PR to be reviewed by @UKiffmeier and to be merged as-is by @pmai next week if no problems crop up. rc.2 to be cut at the end of next week and MA release process to be started once that has happened.

File: schema/SystemStructureCommon.xsd [hunk 1]
```diff
@@ -274,15 +274,15 @@
                                 </xs:documentation>
                             </xs:annotation>
                         </xs:element>
-                        <xs:element name="Signature" type="ssc:SignatureType" minOccurs="0" maxOccurs="unbounded">
+                        <xs:group ref="ssc:GSignature">
                             <xs:annotation>
                                 <xs:documentation xml:lang="en">
                                     This element can contain digital signature information on the meta data referenced
```
Comment (UKiffmeier) [inline schema/SystemStructureCommon.xsd:280]: Minor issue:
This annotation at the instance of the GSignature group is a duplicate of the comment at the group definition (line 349).
If I look at the SSP schema in XML Spy, I see sometimes groups with annotations at the usages of the group, and sometimes not.
image
I think, in this case, the Annotation can be deleted here.

File: docs/4___common_content.adoc [hunk 2]
```diff
@@ -541,16 +539,31 @@ The last option allows the specification of meta data sources that reside inside
...
 
 When this element is present, its contents *MUST* be an `srmd:SimulationResourceMetaData` element as specified by the SRMD schema of the SSP Traceability Specification, if the type attribute of the enclosing MetaData element is `application/x-srmd-meta-data`, or any other valid XML content if the type attribute references another MIME type.
 In that case there *SHOULD* be a layered specification that defines how embedding the content works for that MIME type.
 
+[#ssc_signature]
+==== Signature Sequence
+
+image:images/SystemStructureCommon_GSignature.png[image,width=652,height=133]
+
+The group GSignautre allows the specification of digital signatures for a given model element.
```
Comment (UKiffmeier) [inline docs/4___common_content.adoc:554]: Typo GSignautre -> GSignature

**Review by UKiffmeier — APPROVED** [review]
There are two minor issues regarding a typo and an Annotation. Please check if you want to fix them, or not.
The PR can be completed.

File: docs/4___common_content.adoc (hunk 2 above)
Comment (pmai) [reply docs/4___common_content.adoc:554]:
```suggestion
The group GSignature allows the specification of digital signatures for a given model element.
```

File: schema/SystemStructureCommon.xsd (hunk 1 above)
Comment (pmai) [reply schema/SystemStructureCommon.xsd:280]: I retained the annotation as it is not a full duplicate of the definition annotation, as this puts the relationship to the meta data into the text (and not just any enclosing data), so might be slightly clearer. We might want to revisit our use of annotations going forward in a more consistent style, however keeping it like this for now.

**Pull request merged**